test/helpers.go

```diff
@@ -0,0 +1,97 @@
+package test
+
+import (
+  "fmt"
+  "net/http"
+  "app/jsonHttpHandler"
+)
+
+func getRouteData() []jsonHttpHandler.RouteData {
+  return []jsonHttpHandler.RouteData{
+    jsonHttpHandler.NewRouteData(
+      http.MethodGet,
+      "/resources",
+      func(g jsonHttpHandler.Globals) http.HandlerFunc {
+        return func(w http.ResponseWriter, r *http.Request) {
+          w.WriteHeader(http.StatusOK)
+          fmt.Fprintf(w, getIndexBody)
+        }
+      },
+      []jsonHttpHandler.Middleware{},
```

Comment:
NIT, any reason for using empty slices instead of nil?

Reply:
I see that it works indeed, but I'm wondering what is better. I think in a `sane` language, `nil` and an empty collection are two different things. Not sure I want to rely on quirks of golang.

Reply:
The topic is covered here https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices and in referenced video. I believe in case of Go you're not relying on "an unintential side effect" even by using `nil`.